fieldservice_stock/views/stock.xml

```diff
@@ -13,4 +13,16 @@
               parent="fieldservice.reporting"
               sequence="30"/>
 
+    <menuitem id="menu_fsm_stock_request"
+              name="Stock Request"
+              action="stock_request.stock_request_order_action"
+              parent="fieldservice.operations"
+              sequence="20"/>
+
+    <menuitem id="menu_inv_stock_request"
+              name="Stock Request"
+              action="stock_request.stock_request_order_action"
+              parent="stock.menu_stock_warehouse_mgmt"
+              sequence="25"/>
```

Comment:
@osi-scampbell This should be moved to https://github.com/OCA/stock-logistics-warehouse/tree/12.0/stock_request_picking_type